Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -1602,7 +1623,7 @@
 				ORGANIZATIONNAME = "Wikimedia Foundation";
 				TargetAttributes = {
 					D4991434181D51DE00E6073C = {
-						DevelopmentTeam = D5RR3ZF62D;
+						DevelopmentTeam = SLUNTGN77L;
```

Comment:
You can include them in a `.xcconfig` file and avoid holding any crucial data in the project, it means it can be versioned controlled too.

See:
> http://jamesdempsey.net/2015/01/31/generating-xcode-build-configuration-files-with-buildsettingextractor-xcodeproj-to-xcconfig/